src/app/action-items/action-items.component.ts

```diff
@@ -60,4 +70,11 @@ export class ActionItemsComponent implements OnInit {
         }
       });
   }
+  private getActionItem(): void {
+    this.actionItem = this.tasksService.getActionItem(this.actionItemId);
+  }
+
+  changePath() {
```

Comment:
Could you explain this logic? I think it is not needed since redirecting to the default `/items` ulr is handled in routing configuration.